README.md

```diff
@@ -38,16 +36,14 @@ Make sure you have installed all of the following prerequisites on your developm
   ```bash
   git clone --depth=1 https://github.com/commercetools/nodejs-boilerplate.git
   ```
-  2. Clean up and install
+  2. Clean up, setup and install.  
+  Takes the project name and an optional second directory like `bin` to use.
```

Comment:
It will create the directory and add it to the watch task. If you omit the argument the setup script will still work.